apps/_pomodoro_app/_realtime_tests/cyclicQueue.spec.js

```diff
@@ -0,0 +1,30 @@
+const CyclicQueue = require('../realtime/application/utils/cyclicQueue');
```

Comment:
as for me, it's better to put the spec near the implementation. It's provided to the reader with some knowledge about test coverage.

Reply:
Yes, that seems right